keras/backend.py

```diff
@@ -4430,13 +4431,19 @@ def rnn(step_function,
       zero_output_for_mask: Boolean. If True, the output for masked timestep
           will be zeros, whereas in the False case, output from previous
           timestep is returned.
+      return_all_outputs: Boolean. If True, all outputs of the process will be
```

Comment:
"""
If True, return the recurrent outputs for all timesteps in the sequence.
If False, only return the output for the last timestep (which consumes
less memory).
"""